src/components/CodeEditor.tsx

```diff
@@ -221,7 +227,7 @@ export const CodeEditor = ({
             hoverTooltip((view, pos, side) => {
               const { from, to, text } = view.state.doc.lineAt(pos)
               const line = text.slice(from, to)
-              const match = line.match(/@tsci\/[\w\-.]+/)
+              const match = line.match(/@tsci\/[\w.]+/)
```

Comment:
you reverted some fixes here. Make sure to only commit changes that are relevent for your fix